spec/system/task_spec.rb

```diff
@@ -100,9 +92,8 @@
   describe 'Task削除' do
     context '正常系' do
       # FIXME: テストが失敗するので修正してください
-      xit 'Taskが削除されること' do
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      it 'Taskが削除されること' do
+        task = FactoryBot.create(:task, project_id: project.id, title: 'task_1')
```

Comment:
この記載も省略できる部分は省略しましょう。
```suggestion
        task = create(:task, title: 'task_1')
```

Reply:
5f731f42a2e9dbb9ba959a601d815d20b8f71299 不必要な記載を省略しました